iml-agent/src/rpm.rs

```diff
@@ -1,25 +1,55 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
 use regex::Regex;
 
+use std::fmt;
 use std::process::Output;
+use std::str;
 
 use crate::{agent_error::ImlAgentError, cmd::cmd_output};
 
-fn parse(output: Output) -> Result<bool, ImlAgentError> {
+#[derive(Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct Version(String);
```

Comment:
I wrote version processing for the check_kernel plug-in.  I forgot to push a patch to sperate it out to it's own file.
Please see #1386, it's meant for comparison of rpm versions.

Reply:
Thanks, I'll take a look.

Meanwhile, let's not tie the different changes too tightly for now. Version comparison isn't implemented in this PR, and `Eq` should work ok even treating versions as strings.

I'm dropping derives of `PartialEq` and `Eq` since they're unneeded at the moment with the intent of merging this PR.